### DISCUSSION TITLE
Help with instal GOG to play Cyberpunk 20777

### DISCUSSION THREAD

**Discussion by Thirdys** [Whisky-App/Whisky#488] [Q&A] [answered]
I can't instal GOG ,because it needs .NET. I try to instal it ,but that didn't help. What can I do?

**Comment by mavethee** [comment, marked as answer]
You can use Heroic instead, 

It's not perfect as sometimes it's bugged with game list syncing etc. but at current state somewhat is able to replace Epic, GOG and Prime stores and it's native ARM unlike mentioned apps separately

https://heroicgameslauncher.com/downloads

Eventually `brew install heroic` in Terminal.

Create bottle named `Heroic` (or whatever you want) in Whisky, config it to your likings, e.g bump Windows version to `19042`, enable Retina Mode, Metal HUD etc.

Once it's done, select bottle -> right click -> Show in Finder.

In Finder once it opens correct path to bottle will be selected,
Press `Option` + `Command` + `P` to show path where you're located -> right click on the end of the path -> `Copy [Bottle's UUID] as pathname`

In Heroic once you're logged in, 
- `Settings` -> `Game Defaults` -> `Wine version` -> select `GPTK`,
- `Settings` -> `Game Defaults` -> `Wine prefix` -> paste copied path to Whisky's bottle.

Games by default will be installed to `~/Games/Heroic`, you can change it in `Settings` -> `General` but my advice to to install them inside the bottle.

**Reply by Thirdys** [reply]
Thanks a lot,bro
